SPTDataLoader.podspec

```diff
@@ -13,6 +13,8 @@ Pod::Spec.new do |s|
 
     s.ios.deployment_target = "6.0"
     s.osx.deployment_target = "10.8"
+    s.tvos.deployment_target = '9.0'
+    s.watchos.deployment_target = '2.0'
```

Comment:
AFAIK, CocoaPods doesn't do any introspection from the project file. Some pods have no project file at all. I kind of like that everything has to be explicit in the podspec especially as the pod target is generated procedurally rather than copying over settings from the weirdness that is xcode projects.